packages/wp-now/src/wp-now.ts

```diff
@@ -236,6 +240,28 @@ async function runPluginOrThemeMode(
 	mountMuPlugins(php, documentRoot);
 }
 
+async function runWpPlaygroundMode(
+	php: NodePHP,
+	{ documentRoot, wordPressVersion, wpContentPath, absoluteUrl }: WPNowOptions
+) {
+	const wordPressPath = path.join(
+		getWordpressVersionsPath(),
+		wordPressVersion
+	);
+	php.mount(wordPressPath, documentRoot);
+	await initWordPress(php, wordPressVersion, documentRoot, absoluteUrl);
+
+	fs.ensureDirSync(wpContentPath);
+	fs.copySync(
+		path.join(getWordpressVersionsPath(), wordPressVersion, 'wp-content'),
+		wpContentPath
+	);
```

Comment:
I would prefer the playground mode runs always the same WordPress instance, not based on the current path.

Reply:
Good suggestion! Updated with b661e679

Reply:
Nice!, thank you!